ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
Really interesting discussion!

I think I'm coming down on the side that a component is a component. The underlying technology that comprises the component should, ideally, be of no concern to the developer who is using it. The loader, rather than a generic wrapper component, can provide any common classes (for example)

The implementation is not perfectly seamless as we still have to specify a `block-styles` prop which marks `Logo` as an SVG. That said, I'm not sure if this is really a problem. We ought to decouple components from its underlying technology and content as much as possible, while acknowledging that we might not achieve this all the time.